modules/gitops-argocd-cli-enabling-tab-completion.adoc

```diff
@@ -0,0 +1,41 @@
+// Module is included in the following assemblies:
+//
+// * gitops_cli_argocd/configuring-argocd-gitops-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argocd-cli-enabling-tab-completion_{context}"]
+= Enabling tab completion
+
+After you install the {gitops-shortname} `argocd` CLI, you can enable tab completion to automatically complete `argocd` commands or suggest options when you press Tab.
```

Comment:
I am confused about the wording "press Tab". Is Tab an option that appears in the CLI? I am almost thinking it should be bold like a UI element or in backticks.

Reply:
It's the Tab key on your keyboard. This only applies on the command line (most likely from a terminal application).